packages/restapi/src/lib/spaceV2/start.ts

```diff
@@ -0,0 +1,75 @@
+import { produce } from 'immer';
```

Comment:
please add a short file level comment specifying what this function does. moving forward will add more descriptions of the working here